modules/ggipc-client/src/client.c

```diff
@@ -157,6 +184,15 @@ GglError ggipc_connect_auth(GglBuffer socket_path, GglBuffer *svcuid, int *fd) {
         return GGL_ERR_FAILURE;
     }
 
+    if (common_headers.content_type.data != NULL) {
```

Comment:
We don't use conack payload so content_type doesn't matter (and would be unexpected if present)